.github/workflows/test.yml

```diff
@@ -3,7 +3,10 @@ name: Test
 on:
   pull_request:
     branches:
-      - 'v[0-9]+'
+      - "v[0-9]+"
+    paths-ignore:
+      - "website/**"
+      - "**.md"
```

Comment:
Will this match the readmes in the individual packages? If so, we should avoid those since that will actually change the content of what gets published to npm.

Reply:
Yes it will match those. True that it will change what gets published, but running the tests for a readme change isn't at all connected to publishing, is it?

Reply:
That would be weird wouldn't it!